src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -768,6 +771,10 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
             <ls_deser>-obj     = li_obj.
             <ls_deser>-xml     = lo_xml.
             <ls_deser>-package = lv_package.
+            <ls_deser>-item-abap_language_version = lo_abap_language_vers->get_abap_language_vers_by_objt(
```

Comment:
Let's move this assigment up to right after `ls_item-devclass = lv_package.` (line 108). This way ALaV will be passed properly to the constructor via `create_object`.